mbuild/coordinate_transform.py

```diff
@@ -425,6 +422,8 @@ def translate_to(compound, pos):
         The coordinate to translate the compound to.
 
     """
+    raise ValueError('Please use Compound.translate_to()')
```

Comment:
Since we are throwing an error rather than a warning, I think we should slightly change these messages.  Something like:
``` translate_to() is now deprecated.  Please use Compound.translate_to() ```.  Also might be worth looking up the exact version these functions were deprecated.